engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -466,9 +467,14 @@ case class WorkflowActor(workflow: WorkflowDescriptor)
       val updatedData = startRunnableCalls(data)
       self ! CheckForWorkflowComplete
       stay() using updatedData
-    case Event(CallStarted(callKey), data) if !data.isPending(callKey) =>
+    case Event(CallStarted(callKey, maybeCallLogs), data) if !data.isPending(callKey) =>
       executionStore += callKey -> ExecutionStatus.Running
-      persistStatus(callKey, ExecutionStatus.Running)
+      // TODO: We're not using/waiting for the future results
```

Comment:
Actually you should be able to flip the order of these statements so at least Cromwell wouldn't exit with the call logs write pending.